release-notes/README.md

```diff
@@ -4,6 +4,17 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## July 25, 2022
+
+### ACP React Native Core 2.0.2
+
+* Targeting android 12 (API 31) for the Android implementation
```

Comment:
Android 12 should be a capital A.

Reply:
Also, missing a period at the end of the sentence.

Reply:
The PR is updated.